tests/common/value_operations.h

```diff
@@ -15,24 +15,67 @@
 #include "../../util/type_traits.h"
 
 #include <cassert>
+#include <tuple>
+#include <utility>
+#include <variant>
 
 namespace value_operations {
 
+namespace detail {
+
 template <typename T, size_t N>
 using ArrayT = T[N];
 
+/**
+ * @brief Function allows to make assignemt operations to elements of the
+ * container using std::get and std::index_sequence
```

Comment:
```suggestion
 * @brief Function allows to make assignment operations to elements of the
 * container using std::get and std::index_sequence
```

Reply:
This misprint has been fixed in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/354/commits/7276c4ed3f3f0e1bf0543845082d53a62c3d0ebd